Readme.md

```diff
@@ -4,13 +4,13 @@
 
 ## Overview
 
-[Hastlayer](https://hastlayer.com/) - be the hardware. Automatically transforming [.NET](https://www.microsoft.com/net/) assemblies into computer chips to improve performance and lower power consumption.
+[Hastlayer](https://hastlayer.com/) - Be the hardware. Hastlayer automatically transforms [.NET](https://www.microsoft.com/net/) assemblies into computer chips, improving performance and lowering power consumption.
 
-Hastlayer uses [FPGAs](https://en.wikipedia.org/wiki/Field-programmable_gate_array) (chips that can be "re-wired" on the fly): You just need to select the compute-bound part of your .NET program and Hastlayer will seamlessly swap it out with a generated FPGA implementation. Since not C#, VisualBasic or other code but .NET Intermediate Language assemblies are transformed in theory any .NET language can be used, including C#, VB, F#, C++, Python, PHP, JavaScript...
+Hastlayer uses [FPGAs](https://en.wikipedia.org/wiki/Field-programmable_gate_array) (chips that can be "re-wired" on the fly): The user simply selects the compute-bound part of his .NET program, and Hastlayer seamlessly swaps it out with a generated FPGA implementation. Since .NET Intermediate Language assemblies (but not C#, VisualBasic or other code) are transformed, any .NET languages (e.g.,  C#, VB, F#, C++, Python, PHP, and JavaScript) can be used in theory.

```

Comment:
The language is intentionally talking to the reader with "you", please keep that, otherwise good changes.

Not "in theory any .NET language" (also singular "language")?

Reply:
you are right. I have changed it to the singular form.

Reply:
Please note my remark about using "you" too.

Reply:
Fixed.

Reply:
Would you mind if I edited README files in your other projects?

Reply:
Thanks!

No, please go ahead, that would be great. Keep in mind that most of the time the dev branch contains the latest code so use that to branch your own patch branches out of.

Also, may I suggest this one next? https://github.com/Lombiq/Orchard-Training-Demo-Module This is unrelated to Hastlayer, it's a plugin solely for training purposes made for the Orchard (Core) content management system. This is our open source project where the text is of the most importance since it contains a lot of documentation. There actually the orchard-core branch is the most recent.

Reply:
Thank you for the suggestion!
I have revised the README file on the shared link. Please review them and let me know if you disagree with any of the revisions.